sys/include/ztimer/mock.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2018 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    sys_ztimer_mock ztimer mock clock backend
+ * @ingroup     sys_ztimer
+ * @brief       ztimer mock clock backend
+ *
+ * This ztimer module implements a virtual clock that can be used for unittests.
```

Comment:
```suggestion
 * This ztimer module implements a virtual clock that can be used for unittests.
 * It can be manually adjusted to different timestamps and manually fired to
 * simulate different scenarios and test the ztimer implementation using this as
 * a backing timer
```